packages/react-button/src/components/Button/Button.scss

```diff
@@ -164,6 +64,8 @@
   * 3. Avoid defining styles directly.
  **/
   &._primary {
+    // TODO: Check with David
```

Comment:
@dzearing - would need to pick your brain on this one. this would mean another v8 breaking related to theming buttons, but i guess we have to just accept that and document this well?